src/Styling/Home.css

```diff
@@ -128,6 +128,101 @@
   width: 44%;
 }
 
+/* Project section */
+.Project_section {
+  margin-top: 75px;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.Project_section a {
+  text-decoration: none;
+}
+
+.Project_style_1 {
+  margin: 40px auto;
+  width: 80%;
+  border: none;
+}
+
+.Project_style_1 h2 {
+  font-size: 1.6rem; 
+  width: 180%;
+}
+
+.Project_style_1 p {
+  text-align: right;
+  margin-top: 20px;
+  font-size: 13px;
+}
+
+.Project_style_1 img {
+  width: clamp(8rem, 80vw, 40rem);
+  height: 300px;
+  background-color:rgba(0,0,0,0.6);
+}
+
+.Container {
+  display: flex;
+  justify-content: space-around;
+  flex-wrap: wrap;
+}
+
+.Container_2 {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
+
+.Project_style_2 {
+  display: flex;
+  width: 40%;
+  margin: 40px auto;
+  flex-wrap: wrap;
+}
+
+.Project_style_2 h2 {
+  font-size: 1.2rem; 
+  width: 49%;
+}
+
+.Project_style_2 p {
+  margin-top: -10px;
+  font-size: 13px;
+}
+
+.Project_style_3 {
```

Comment:
I think you’re overcomplicating the styles needed for this section. Yes, you have 4 projects in the mockup, but you only have two possible variations of styles for each project(the project is either the full width of the screen, or half the width). you dont need a different style for all 4 projects. Go back to the mockup, take a moment to notice the similarities and identify which styles can be combined  and made simpler.